Plugin Cubot/src/main/java/net/simon987/cubotplugin/event/ExecutionTimeEvent.java

```diff
@@ -0,0 +1,23 @@
+package net.simon987.cubotplugin.event;
+
+import net.simon987.server.event.GameEvent;
+import net.simon987.server.game.objects.GameObject;
+
+public class ExecutionTimeEvent extends GameEvent {

```

Comment:
You can remove the execution event and its listener. Instead create a function that is called in this loop. There, `cost` is the real time execution, which you can add to the user's total execution time (Without creating an event + a listener)

https://github.com/simon987/Much-Assembly-Required/blob/e378016c566f546d86eebb28892746a7a3fef5e7/Server/src/main/java/net/simon987/server/GameServer.java#L168-L184

Sorry for the confusion, I should have looked into this part a bit more before asking you to make a event listener for this.

Reply:
I did what you said I'm not sure if the function works but it is setup and I seem to not have any errors